src/physics/large_scale_condensation.jl

```diff
@@ -78,7 +78,7 @@ function large_scale_condensation!(
 
     (;gravity, water_density) = constants
     (;Δt_sec) = time_stepping
-    pₛΔt_gρ = pₛ*Δt_sec/gravity/water_density   # precompute constant
+    pₛΔt_gρ = pₛ*value(Δt_sec)/gravity/water_density   # precompute constant
```

Comment:
`.value` here too, or via implementing *,/ as denoted in comment in PR

Reply:
addressed with [fd2b5e2](https://github.com/SpeedyWeather/SpeedyWeather.jl/pull/418/commits/fd2b5e24547033dd18cffaca922ab7b9f559ad6f)